sql/common.go

```diff
@@ -59,21 +59,6 @@ func createSpan(ctx context.Context, tracer *go2sky.Tracer, opts *options, opera
 	return s, nil
 }
 
-func createLocalSpan(ctx context.Context, tracer *go2sky.Tracer, opts *options, operation string) (go2sky.Span, context.Context, error) {
```

Comment:
No local span? Why it was there before?

Reply:
this because we treat a transcation span as a local span before, each operation in transcation is the child span of this local span, now it is not needed

Reply:
Got it.